scripts/http_client_notification.py

```diff
@@ -0,0 +1,42 @@
+import requests
+import json
+from loguru import logger
+from websocket_client import get_jwt_access, FORUM_HOST, FORUM_PORT
+
+
+def get_notifications(access_token):
+    response = requests.get(
+        f"http://{FORUM_HOST}:{FORUM_PORT}/notifications/",
+        headers={"Authorization": f"Bearer {access_token}"},
+        cookies={"access_token": access_token}
+    )
+    if response.status_code == 200:
+        logger.info("Notifications:")
+        logger.info(json.dumps(response.json(), indent=4))
+        return response.json()
+    else:
+        logger.error(f"Failed to get notifications: {response.status_code}")
```

Comment:
If the response status code is not 200, the error handling does not log or handle the response content.

Reply:
Fixed, thanks